modules/installation-generate-aws-user-infra-install-config.adoc

```diff
@@ -63,18 +67,6 @@ ifdef::openshift-origin[]
 This field is optional.
 endif::[]
 
-. Edit the `install-config.yaml` file to set the number of compute replicas, which are also known as worker
```

Comment:
I wanted to add a brief explanation as to why we ask the reader to do this. But, after testing, I have removed this step because:

* It does not prevent the compute node manifests from being created. Those are later deleted in the 'Creating the 
Kubernetes manifest and Ignition config files' section.

* It results in `mastersSchedulable` being set to `true`, which is then later reversed in the 'Creating the Kubernetes manifest and Ignition config files' section. So it seems that we ask the reader to set something, then unset it later.

1. @kalexand-rh mentioned that this step might have been introduced into the docs because of a bug. Does anyone have a reference to that BZ so that we can determine if it has been resolved?

2. Does anyone know of any reason why this step should not be taken out? I tested the full AWS UPI installation without setting it and it worked for me.

Reply:
@pneedle-rh 

> It does not prevent the compute node manifests from being created. Those are later deleted in the 'Creating the Kubernetes manifest and Ignition config files' section.

right, in UPI, the master and worker replica numbers depends on CF template.
   

> It results in mastersSchedulable being set to true, which is then later reversed in the 'Creating the Kubernetes manifest and Ignition config files' section. So it seems that we ask the reader to set something, then to reverse that later.

Currently:
compute.replicas: 0 -> mastersSchedulable: true
otherwise -> mastersSchedulable: false

I saw in step 4 of `Creating the Kubernetes manifest and Ignition config files`, user is asked to check `mastersSchedulable` parameter, I think it's good and necessary for UPI installation.


> @kalexand-rh mentioned that this step might have been introduced into the docs because of a bug. Does anyone have a reference to that BZ so that we can determine if it has been resolved?

@kalexand-rh  may help on this.

> Does anyone know of any reason why this step should not be taken out? I tested the full AWS UPI installation without setting it and it worked for me.

Right, per above answers, edit compute.replicas is not necessary for UPI installation.

Reply:
@yunjiang29 thank you for clarifying this, this is very helpful. Based on your comments, I will continue with the step removed.